packages/e2e-tests/testFixture.ts

```diff
@@ -23,30 +22,16 @@ const testWithCloneRecording = base.extend<TestIsolatedRecordingFixture>({
   exampleKey: undefined,
   testUsers: undefined,
   pageWithMeta: async ({ page, exampleKey, testUsers }, use) => {
-    const exampleRecordings = require("./examples.json");
```

Comment:
Why were we using `require` here before?

Using `import` gives up TSC hints for the JSON structure.